docs/source/gbd2017_models/risk_attributable_causes/iron_deficiency/index.rst

```diff
@@ -509,7 +560,9 @@ and age groups.
 Because we are not modeling individual causes of anemia (and their associated 
 mortality), we assume that all simulants have the same mortality rate regardless 
 of their hemoglobin value, when in reality, those with lower hemoglobin values 
-will have higher mortality rates than those with higher hemoglobin values.
+will have higher mortality rates than those with higher hemoglobin values. 
+Notably, deaths due to causes with iron responsive anemia sequelae account for 
+approximately 1.1% of deaths in the first five years of life globally. 
```

Comment:
@alibow It would be good to include a reference for this number. For example, you could put your notebook in the vivarium data analysis repo and make a link to it, or add a link to GBD compare.